### DISCUSSION TITLE
Console drag to resize is lagging alot after adding tailwind to the project.

### DISCUSSION THREAD

**Discussion by WingsDevelopment** [tailwindlabs/tailwindcss#9820] [Help]
Hello, I am not sure if this is bug, or strange behavior, or something else... But when i drag my console to resize it, it lags alot. I've just added tailwind package, tailwind config & updated my index.css file. And after removing tailwind, everything comes back to normal...

![image](https://user-images.githubusercontent.com/24592343/201489611-dcb09c4a-ee89-45d7-a390-0fc18d74a8e3.png)

I've created react app with vite, and using chrome.
These are versions i installed:
    "postcss": "^8.4.19",
    "tailwindcss": "^3.2.4",

**Comment by adamwathan** [comment]
Hey! Can you share an example project that reproduces the issue?

**Reply by WingsDevelopment** [reply]
Well I've just created react 18 with vite,
https://www.digitalocean.com/community/tutorials/how-to-set-up-a-react-project-with-vite
And then I've added tailwind following this steps:
https://tailwindcss.com/docs/guides/create-react-app
That's all you need. 
Mozila seems to work fine, but chrome is really bugging...
I have lorem1000 text and 8 images, but i think it will lag even with empty page..

**Reply by adamwathan** [reply]
If you can create a GitHub repository we can clone that reproduces the issue we can take a look 👍🏻 unfortunately we receive too many questions to be able to recreate every problem ourselves based on just a description so we rely on reproductions to be able get through all of the questions we get with our small team.

**Reply by WingsDevelopment** [reply]
I understand, here you go sir https://github.com/WingsDevelopment/react18-tailwind-test
